src/bun.js/webcore/response.zig

```diff
@@ -172,7 +178,7 @@ pub const Response = struct {
     }
 
     pub fn isOK(this: *const Response) bool {
-        return this.body.init.status_code == 304 or (this.body.init.status_code >= 200 and this.body.init.status_code <= 299);
+        return this.init.status_code == 304 or (this.init.status_code >= 200 and this.init.status_code <= 299);
```

Comment:
Unrelated to your changes, but might as well fix it. `response.ok` is only suppose to be true for `[200,300)`. This was a mistake when implemented initially.

Reply:
done